app/repositories/UIJourneySessionDataRepository.scala

```diff
@@ -105,4 +105,15 @@ class UIJourneySessionDataRepository @Inject()(
       .deleteOne(dataFilter(data))
       .toFuture()
       .map(_ => true)
+
+  private def sessionFilter(sessionID: String): Bson = {
+    import Filters._
+    and(equal("sessionId", sessionID))
+  }
+  
+  def deleteOne(sessionId: String): Future[Boolean] =
```

Comment:
* this method is only use in scope of the Spec/tests;
* you should be able to use existing method instead, have a look at example in: => UIJourneySessionDataISpec

```
    "deleteOne should remove a sessionData item" in {
      await(repository.set(UIJourneySessionData("session-12345", "ADD-UKPROP", Some(AddIncomeSourceData(Some("business1"))))))
      val result = await(repository.deleteOne(UIJourneySessionData("session-12345", "ADD-UKPROP")))
      result shouldBe true
      val sessionData = await(repository.get("session-12345", "ADD-UKPROP"))
      sessionData shouldBe None
    }
```

Reply:
This was Solar's implementation for clearing the data between tests. You use this in the beforeAll() and supply the sessionID which clears all data as opposed to the existing method requires a data: UIJourneySessionData which is annoying to use in

Reply:
* it is not correct to add business logic just to satisfy tests scenario;
 * using repo and call required collection method (delete should be fine);